pkg/healthcheck/healthcheck.go

```diff
@@ -1439,6 +1439,9 @@ func CheckProxyVersionsUpToDate(pods []corev1.Pod, versions version.Channels) er
 			}
 		}
 	}
+	if versions.Empty() {
+		return fmt.Errorf("unable to determine version channel")
```

Comment:
nit: wdyt of `errors.New`, we don't need to actually format anything here it seems. The healthcheck package seems to [import `errors`](https://github.com/linkerd/linkerd2/blob/dc6d1e45d49e87ac72b444602bd972b89bab79d3/pkg/healthcheck/healthcheck.go#L7) too.